### DISCUSSION TITLE
Firmware install connection issues

### DISCUSSION THREAD

**Discussion by MrCrazyGoose** [pybricks/support#1618] [General]
When trying to install the PyBricks firmware on a SPIKE hub, I am able to get into update mode (with the flashing colored lights on the Bluetooth button), but the SPIKE hub does not show up in the connection window, nor does it show up in the device manager. I tried it on three different computers, but we lost the cord a while ago, so I am just using some cord that fits, which may be an issue (I also tried multiple cords).

**Comment by dlech** [comment]
We had a similar discussion at https://github.com/orgs/pybricks/discussions/1608 recently. Does any of that help?

**Reply by MrCrazyGoose** [reply]
I tried most of the stuff here, but I am ordering a new cord anyway (the actual one that comes with the SPIKE). It seems that for this person, it just magically worked at some point when trying with a different computer, which I already did twice, but it is good to know that the issue was solvable for someone else.

**Reply by dlech** [reply]
I was mainly wondering if it was showing up as an "Unknown USB device" like in https://github.com/orgs/pybricks/discussions/1608#discussioncomment-9191400 or if windows just doesn't detect it at all (i.e. you are using a charging only cable that doesn't have data wires).

**Reply by MrCrazyGoose** [reply]
Yeah, I think that could be the issue. There is no "Unknown USB device," but it could be under another name. I just ordered a new cable so hopefully that will fix it.

**Reply by MrCrazyGoose** [reply]
It was the cord. I got a new one, and it worked. Because of the different cord, it was probably under another name, but the issue is resolved.
